enhancements/sig-architecture/15-resourcebasedscheduling/README.md

```diff
@@ -0,0 +1,191 @@
+	// with larger allocatable to capacity ratio of a set of cluster resources;
+	ClusterResourcePreferenceTypeMostAllocatableToCapacityRatio ClusterResourcePreferenceType = "MostAllocatableToCapacityRatio"
+	// ResourcePreferencePolicyTypeMostAllocatable, the placement favors clusters with larger
+	// amount of allocatable resources;
+	ClusterResourcePreferenceTypeMostAllocatable ClusterResourcePreferenceType = "MostAllocatable"
+)
+
+type ClusterResource struct {
+	// ResourceName represents the name of the cluster resource
+	ResourceName ClusterResourceName `json:"resourceName"`
+}
+
+type ClusterResourceName string
+
+const (
+	ClusterResourceNameNameCPU ClusterResourceName = "cpu"
+	ClusterResourceNameMemory  ClusterResourceName = "memory"
+)
+```
+
+#### 2. Creating a prioritizer, `mostallocatabletocapacityratio`, to score with allocatable to capacity ratio of cluster resources.
+
+#### 3. Creating another prioritizer, `mostallocatable`, to score with the allocatable of cluster resources.
+
+
+### Examples
+
+#### 1. Select 3 clusters whose CPU/memory usage is less than others. (Use Story 1)
+
+```yaml
+apiVersion: cluster.open-cluster-management.io/v1alpha1
+kind: Placement
+metadata:
+  name: placement1
+  namespace: ns1
+spec:
+  numberOfClusters: 3
+  clusterResourcePreference:
+    clusterResources:
+      - resourceName: cpu
+      - resourceName: memory
+```
+
+#### 2. Select a cluster with the largest memory available. (Use Story 2)
+
+```yaml
+apiVersion: cluster.open-cluster-management.io/v1alpha1
+kind: Placement
+metadata:
+  name: placement2
+  namespace: ns1
+spec:
+  numberOfClusters: 1
+  clusterResourcePreference:
+    type: MostAllocatable
+    clusterResources:
+      - resourceName: memory
+```
+
+## Design Details
+
+### Resource data required during scheduling
+Currently the following resource data is available for both CPU and Memory in the status of ManagedCluster, which is updated by registration agent.
+- `allocatable`, sum of the `allocatable` on each node in a managed cluster;
+- `capacity`, sum of the `capacity` on each node in a managed cluster;
+
+```yaml
+apiVersion: cluster.open-cluster-management.io/v1
+kind: ManagedCluster
+metadata:
+  name: local-cluster
+spec:
+  hubAcceptsClient: true
+  leaseDurationSeconds: 60
+status:
+  allocatable:
+    cpu: '15'
+    memory: 63194Mi
+  capacity:
+    core_worker: '18'
+    cpu: '18'
+    memory: 69938Mi
+    socket_worker: '6'
+```
+In order to support other resources, like GPU, the `allocatable` and `capacity` should be included in the status of managed cluster either.
+
+### Plugin `mostallocatabletocapacityratio`
+It is a pritoritizer and scores feasible managed clusters with the process below.
+- If the placement has `ClusterResourcePreference` specified in the spec and its `Type` is `MostAllocatableToCapacityRatio`, the score of a managed cluster is the sum of the score for each resource.  
+`score = sum(resource_x_allocatable / resource_x_capacity))`  
```

Comment:
it is worth to  mention that  we treat each  resource dimension with equal weight and the reasoning.

Reply:
Updated.